contracts/note/src/msg.rs

```diff
@@ -18,6 +21,7 @@ pub enum ExecuteMsg {
     /// a callback of Vec<QuerierResult>, or ACK-FAIL if unmarshalling
     /// any of the query requests fails.
     Query {
+        on_behalf_of: Option<String>,
```

Comment:
queries get executed by the voice module (not proxy), so could we remove this field and then when unpacking in contract.rs just set it to `None`? on behalf of doesn't make a ton of sense in this context imo.